ext/drx/drx_buf.c

```diff
@@ -752,22 +751,35 @@ drx_buf_insert_buf_memcpy(void *drcontext, drx_buf_t *buf, instrlist_t *ilist,
         ok = drx_buf_insert_buf_store(drcontext, buf, ilist, where, dst, DR_REG_NULL,
                                       src_opnd, opsz, 0);
         DR_ASSERT(ok);
-        /* update buf pointer, so client does not have to */
-        drx_buf_insert_update_buf_ptr(drcontext, buf, ilist, where, dst, src, len);
     }
+    /* update buf ptr, so that client does not have to */
+    drx_buf_insert_update_buf_ptr(drcontext, buf, ilist, where, dst, src, len);
```

Comment:
Made this change for symmetry; it's also useful to have this here in the case that we want to change the behavior of drx_buf_insert_buf_memcpy() in the future to not internally update the buffer pointer to be more consistent with the other drx_buf methods.